ts/webui/src/static/model/trial.ts

```diff
@@ -234,6 +234,12 @@ class Trial implements TableObj {
         } else {
             const tempHyper = this.info.hyperParameters;
             let params = JSON.parse(tempHyper[tempHyper.length - 1]).parameters;
+            // for hpo experiment: search space choice value is None, and it shows null
+            for (const key in params) {
```

Comment:
Use for-of if there is not special reason.

Reply:
yes!